pkgs/development/python-modules/aio-pika/default.nix

```diff
@@ -0,0 +1,54 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pythonOlder
+, pytestCheckHook
+, aiormq
+, yarl
+, aiomisc
+, shortuuid
+}:
+
+buildPythonPackage rec {
+  pname = "aio-pika";
+  version = "8.3.0";
+
+  disabled = pythonOlder "3.7";
+
+  src = fetchFromGitHub {
+    owner = "mosquito";
+    repo = pname;
+    rev = version;
+    sha256 = "CYfj6V/91J7JA8YSctG/FkSHRkwyLKxr27eREbA+MtQ=";
+  };
+
+  propagatedBuildInputs = [
+    aiormq
+    yarl
+  ];
+
+  checkInputs = [
+    pytestCheckHook
```

Comment:
`pytestCheckHook` should be in `nativeCheckInputs`

Reply:
ty